datasets/ONS-Trade-in-goods/exports.py

```diff
@@ -1,24 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
 #!/usr/bin/env python
 # coding: utf-8
 # %% [markdown]
 # # Country by commodity exports
 
+
 # %%
+
+# In[2]:
+
+
+import pandas as pd
+import numpy as np
 from gssutils import *
 import json
+from gssutils.metadata import THEME
+from gssutils.metadata import *
```

Comment:
import * includes import THEME, so you shouldn't need both of these. I _think_ just the import THEME line is needed, but you'll probably want to confirm.